spec/takeaway_spec.rb

```diff
@@ -1,4 +1,10 @@
 require 'takeaway'
+require 'menu'
 
 describe Takeaway do
+  it 'shows the list of dishes with prices' do
+    food_person = Menu.new
+    food_person.list
+    expect(subject.show_menu).to eq food_person.food_list
```

Comment:
this expectation could benefit from more appropriate variable names such that it reads like a sentence